test/Location.cpp

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
This file should not be here

Reply:
Removing them results in error

Reply:
I explained why you should not leave this file here in the email...

Reply:
I got it. Will make changes in CMakeLists.